client/views/actions/whois.tpl

```diff
@@ -3,12 +3,56 @@
 	<i class="hostmask">({{whois.user}}@{{whois.host}})</i>:
 	<b>{{whois.real_name}}</b>
 </div>
+{{#if whois.actuallhost}}
+<div>
+	Actual host
+	{{> ../user_name nick=whois.nick}}
+	<i class="hostmask">({{whois.user}}@{{whois.actuallhost}})</i>:
+	<b>{{whois.real_name}}</b>
```

Comment:
There's no reason to display real name here again.

Reply:
Figured I'd use the same formatting as for regular host

Reply:
That's just duplicate information.

Reply:
I'll change it if you feel strongly about it, but I don't think that duplicate information has to be a bad thing. In this case it doesn't take more space (the line would be used anyway) and just makes it look somewhat more presentable imho.

Anyway, I don't feel that strongly about it so if needs to be changed I'll change it.